src/modules/app/actions/listen-to-updates.js

```diff
@@ -97,7 +97,7 @@ export function listenToUpdates(history) {
           dispatch(updateOutcomePrice(log.marketId, log.outcome, new BigNumber(log.price, 10)))
           dispatch(updateMarketCategoryPopularity(log.market, log.amount))
           const { address } = getState().loginAccount
-          if (log.sender === address || log.owner === address) {
```

Comment:
This value gets emitted as 'creator' not 'orderCreator' (it actually differs from DB in this way, translated in the object)

https://github.com/AugurProject/augur-node/blob/6c5dbcf45093a123dd0002f08f6f7f9aeb3c6741/src/blockchain/log-processors/order-filled/index.ts#L54

Reply:
Good catch! I'll update this.